src/main/java/frc/robot/classes/Kinematics.java

```diff
@@ -98,17 +98,11 @@ public double distanceFromPose(Position2D checkpoint) {
     double y1 = m_currentPose.getY();
     double x2 = checkpoint.getX();
     double y2 = checkpoint.getY();
-    double distance = Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));
-
-    return distance;
+    return Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));
   }
 
   public boolean atPose(Position2D checkpoint, double threshold) {
     double distanceFromPose = distanceFromPose(checkpoint);
-    if (distanceFromPose <= threshold) {
-      return true;
-    } else {
-      return false;
-    }
+    return distanceFromPose <= threshold;
```

Comment:
Good fix. Add `( )`'s around `distanceFromPose <= threshold` for readability.

Reply:
Done